NAMESPACE

```diff
@@ -5,6 +5,7 @@ export(clean_anzsco)
 export(clean_anzsic)
 export(clean_asced_foe)
 export(clean_asced_qual)
+export(clean_nfd)
```

Comment:
What do you think about naming the function `clean_anzsco_nfd`?

Reply:
Can do! I thought "nfd" was in the anzsic data as well - but I just checked anzsic2006 and didn't find any columns with "nfd" strings. 

Is there any type of dataset which may use this function outside of anzsco? If not I'll change it :)

Reply:
I guess ASCED uses the ", nfd" style in the same way -- so good idea, keep it as is!